HLTrigger/JetMET/src/HLTMETCleanerUsingJetID.cc

```diff
@@ -97,10 +93,10 @@ void HLTMETCleanerUsingJetID::produce(edm::Event& iEvent, const edm::EventSetup&
         double mex_diff = mex_goodJets - mex_jets;
         double mey_diff = mey_goodJets - mey_jets;
         //double sumet_diff = sumet_goodJets - sumet_jets;  // cannot set sumet...
-        reco::Candidate::LorentzVector p4_diff(mex_diff, mey_diff, 0, sqrt(mex_diff*mex_diff + mey_diff*mey_diff));
+        reco::Candidate::LorentzVector p4_diff(met->front().px()+mex_diff, mey_diff+met->front().py(), 0, sqrt((met->front().px()+mex_diff)*(met->front().px() +mex_diff)+(met->front().py()+mey_diff)*(met->front().py() +mey_diff)));
 
-        reco::CaloMET cleanmet = met->front();
-        cleanmet.setP4(cleanmet.p4() + p4_diff);
+        reco::CaloMET cleanmet ;
+        cleanmet.setP4(p4_diff);
```

Comment:
Hi,

I do not see any issue here, it is about renaming a collection that I do not think make any differences at the end in physiscs.

The second part is not clear for me which information will be missing, but in general the idea here is to have a new collection that does not inherent from original caloMET collection